tests/sources/fixtures/oracle/fixture.py

```diff
@@ -62,6 +63,14 @@ def inject_lines(table, cursor, lines):
         print(f"Inserted batch #{batch} of {batch_size} documents.")
 
 
+@retryable(retries=7, interval=2, strategy=RetryStrategy.EXPONENTIAL_BACKOFF)
+async def wait():
```

Comment:
Where is this being called though? Are we not passing it as a Pytest fixture, just rely on fixtures/fixture:main?

Reply:
This code is not called by `pytest` at all - functional tests are facilitated with ftesh.sh file and fixture.py only